rstudio/Dockerfile

```diff
@@ -40,32 +40,80 @@
 #
 ############################################################################
 
+FROM ubuntu:22.04
```

Comment:
Wouldn't it make sense to build this on top of an R ML Runtime image so you don't have to take care of R installs and etc.

Reply:
IIUR the recent R images have Python already installed too, so that can be skipped too.

Reply:
I've used PBJ approach because I think it is more flexible. I little bit more complex but not so much. When I created the image R4.3.2 was not available as a runtime.